src/redux/sagas/actions/manageReputation.ts

```diff
@@ -42,7 +31,7 @@ function* manageReputationAction({
       ? 'emitDomainReputationPenalty'
       : 'emitDomainReputationReward';
 
-    if (!userAddress) {
+    if (!walletAddress) {
       throw new Error(`User address not set for ${batchKey} transaction`);
     }
```

Comment:
I don't this these fail safes are that relevant anymore, as the sagas are pretty protected from incorrect data.

Although this is not the time for refactoring this, so it can stay as it is.